cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/PersistedCassandraFrameworkConfiguration.java

```diff
@@ -145,6 +148,13 @@ public void frameworkId(@NotNull final String frameworkId) {
     }
 
     @NotNull
+    public CassandraFrameworkProtos.RackDc getDefaultRackDc() {
+        CassandraFrameworkProtos.RackDc rackDc = get().getDefaultRackDc();
+        if (rackDc == null) rackDc = CassandraFrameworkProtos.RackDc.newBuilder().setRack("RACK0").setDc("DC0").build();
```

Comment:
Please add `// TODO: Persistence Schema Update` so that we can more appropriately address the code needed to update object schemas in the future.